### PR TITLE
Improve localization

### PR DESCRIPTION
**Pull request by limpbrains**
TLDR:
- localize everything
- convert lang files to json
- remove untranslated eng strings from lang files
- 2 levels deep lang files, react-localization doesn't support more

Now it is very difficult for a developer to write multilingual code. Each time a new expression is added, you need to edit all 25 language files manually. As a result, it’s much easier not to do this, and just to write in English

To make the programmer's job easier, I suggest editing English only. If the phrase is missing in other languages, the English value will be used. 

For this to work, I removed the 3rd level in language files, because react-localization works only with 2 levels.

I converted the language files to JSON so that we don't need to edit them manually. We need to upload them to an online translator as they do in the [bitcoin](https://github.com/bitcoin/bitcoin/#translations). In this case you don't need to know JS to translate BW. We even can automate updating languages using [transifex github action](https://github.com/marketplace/actions/github-transifex-actions)

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/37kgrf7d9wtr0kcrpvnnvjgqw0

[download apk](http://filestorage.bluewallet.io:1488/1594897477.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/13qzv1av6f4etbypmu6bxmamg8

[download apk](http://filestorage.bluewallet.io:1488/1594901715.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kw6z4uj62ajz4d7f2qda8uh054

[download apk](http://filestorage.bluewallet.io:1488/1594908445.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/au9q7nnbm6y87tuwf013tzg31r

[download apk](http://filestorage.bluewallet.io:1488/1594910745.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/k3gkbmy1pn663tpkw5gmecn3pc

[download apk](http://filestorage.bluewallet.io:1488/1594981873.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vhqhdk6ny6dg86h0awz7pxx7mg

[download apk](http://filestorage.bluewallet.io:1488/1594992099.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4cckbe6g3x6tvt6uk0nbcnxgwm

[download apk](http://filestorage.bluewallet.io:1488/1595066034.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dyxry882aq8fa78j74yzux3bvm

[download apk](http://filestorage.bluewallet.io:1488/1595237440.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/e3b1rq3qjpa1az73zvkzgcf720

[download apk](http://filestorage.bluewallet.io:1488/1595241299.apk)

File: screen/wallets/hodlHodlViewOffer.js [hunk 1]
```diff
@@ -221,12 +232,13 @@ export default class HodlHodlViewOffer extends Component {
...
-                        '%' +
-                        ' / ' +
-                        this.state.offerToDisplay.trader.trades_count +
-                        ' trades'
-                      : 'No rating'}
+                      ? loc.formatString(loc.hodl.item_rating, {
+                          rating:
+                            Math.round(this.state.offerToDisplay.trader.rating * 100) +
+                            '% / ' +
+                            this.state.offerToDisplay.trader.trades_count,
```
Comment (Overtorment) [inline screen/wallets/hodlHodlViewOffer.js:239]: `' trades'`  lost

File: tests/unit/loc.test.js [hunk 2]
```diff
@@ -77,46 +76,9 @@ describe('Localization', () => {
...
       currency._setExchangeRate('BTC_USD', 1);
       currency._setPreferredFiatCurrency(FiatUnit.USD);
-      const actualResult = loc.formatBalance(balance, toUnit, withFormatting);
+      const actualResult = formatBalance(balance, toUnit, withFormatting);
       assert.strictEqual(actualResult, expectedResult);
     },
     240000,
   );
-
-  it('has all keys in all locales', async () => {
```
Comment (Overtorment) [inline tests/unit/loc.test.js:86]: since we now have proper json files, we need a simple script that gets all keys from EN and ports them to other lands if the key is missing there.

the point of this is to always have somewhat up-to-date ready-to-translate files: sometimes users who cant even clone git repo come and do a translation, sending resulting files over email. its for them we always maintained a rule to always have all keys in all locales.

File: screen/wallets/hodlHodlViewOffer.js (hunk 1 above)
Comment (limpbrains) [reply screen/wallets/hodlHodlViewOffer.js:239]: Nope it's inside `loc.hodl.item_rating`

File: tests/unit/loc.test.js (hunk 2 above)
Comment (limpbrains) [reply tests/unit/loc.test.js:86]: @Overtorment How about try to use transifex to create nice experience to people who wants to contribute? Can I create an account on transifex in behalf of BW? and set it up?

Comment (Overtorment) [reply tests/unit/loc.test.js:86]: fuck it. lets try it. go ahead
